app-editors/vscode/vscode-1.65.2.ebuild

```
@@ -119,4 +118,12 @@ pkg_postinst() {
 	xdg_pkg_postinst
 	elog "You may want to install some additional utils, check in:"
 	elog "https://code.visualstudio.com/Docs/setup#_additional-tools"
+
+	if has_version -r ">=gui-libs/wlroots-0.15"; then
+		elog "------------------------------------------------------------------"
```

Please use empty line instead of those symbols. I think it does a successful message pass and paragraph break, with less "spam" on screen and log file.

```suggestion
		elog
```

---

Done!